src/index.html

```diff
@@ -7,12 +7,54 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="/src/styles/index.scss"
     />
   </head>
+
   <body>
     <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="MacBook"
+        />
+      </div>
+      <p class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__kod">Product code: 195434</p>
```

Comment:
According to the checklist, classes should be named using the BEM methodology. The class name 'card__kod' seems to be a typo or incorrect naming. If it is meant to represent the product code, consider renaming it to 'card__code' for clarity and consistency with BEM naming conventions.